docs/pages/versions/unversioned/guides/configuring-statusbar.md

```diff
@@ -4,6 +4,9 @@ title: Configuring StatusBar
 
 Expo and React Native provide APIs and configuration options for Android to configure the status bar for your app. These can be used to control the appearance of the status bar in your app.
 
+## Installation
```

Comment:
I don't think we need an `Installation` section in these docs. People can refer to the Constants API docs for the relevant installation instructions